src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -1356,6 +1356,14 @@ class MacroAssembler: public Assembler {
     vmfle_vv(vd, vs1, vs2, vm);
   }
 
+  inline void vmsltu_vi(VectorRegister Vd, VectorRegister Vs2, int32_t imm, VectorMask vm = unmasked) {
```

Comment:
Seems this function is not used in the code?
And, when `imm` == 0, seems it will output unexpected value?

Reply:
We can keep it as it's a good complement to existing methods and the cost is very low. Good point on `imm == 0`, we should probably have an assert for that case or have a `splat(false)` operation.

Reply:
Fixed